MetricsDashboard.Core/Models/BooleanTile.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using MetricsDashboard.Contract;
+using MetricsDashboard.Contract.Enums;
+using MetricsDashboard.Core.Entities;
+
+namespace MetricsDashboard.Core.Models
+{
+    public class BooleanTile : ITile
+    {
+        private readonly string _name;
+
+        private readonly MetricEntity<bool> _latest;
+
+        public BooleanTile(string name, IEnumerable<MetricEntity<bool>> metrics)
+        {
+            _name = name;
+            _latest = metrics.OrderByDescending(m => m.AddedOn).First();
+        }
+
+        public TileDto ToDto()
```

Comment:
I would recommended to use some kind of mapper or ConverterClass to do make convention between types.

Reply:
created TileMapper class